### PR TITLE
Revert "[connectivity_platform_interface] Add `ConnectivityResult.unknown`."

### PR DESCRIPTION
**Pull request by ditman**
Reverts flutter/plugins#2560

### CODE REVIEW

**Comment by ditman** [comment]
(This might need to be rolled back to "1.0.3+1" if we want to merge this fully green, even after "2.0.0" is removed from pub.dev)

**Comment by jonasfj** [comment]
`2.0.0` should now be removed from pub.dev

**Comment by ditman** [comment]
This change will always have a red pre-submit, since the version script checks against `master` and not what's published in pub.dev. Post-submit, the repo should go green again.

**Comment by collinjackson** [comment]
OK, sounds good.

File: packages/connectivity/connectivity_platform_interface/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-## 2.0.0
+## 1.0.3+1
```
Comment (amirh) [inline packages/connectivity/connectivity_platform_interface/CHANGELOG.md]: We don't need to publish a patch bump

Comment (ditman) [reply packages/connectivity/connectivity_platform_interface/CHANGELOG.md]: Done

**Comment by ditman** [comment]
Merging on red, since most of the failures are related to versioning, which is what we're trying to straighten out here.

```
Running version check for changed packages
packages/connectivity/connectivity_platform_interface/pubspec.yaml incorrectly updated version.
HEAD: 1.0.3, master: 2.0.0.
Allowed versions: {3.0.0: NextVersionType.BREAKING_MAJOR, 2.1.0: NextVersionType.MINOR, 2.0.1: NextVersionType.PATCH}
```

**Pull request merged**